recipes/phonopy/COPYING

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2014, Atsushi Togo
```

Comment:
I will ask. But it will probably be included in the next release. 
The license is specified at the webpage (https://atztogo.github.io/phonopy/):
New BSD from version 1.3. (LGPL from ver. 0.9.3 to version 1.2.1., GPL to version 0.9.2.)
Is this a show-stopper @jakirkham ?